spec/controllers/categories_controller_spec.rb

```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe CategoriesController, type: :controller do
+
+end
```

Comment:
Did they teach you how to test using capybara? Also, create an RSPEC testing for the model.